common/lib/dependabot/package_manager.rb

```diff
@@ -64,12 +67,23 @@ def deprecated?
     #   package_manager.unsupported? #=> false
     sig { returns(T::Boolean) }
     def unsupported?
-      return true if unsupported_versions.include?(version)
+      false
```

Comment:
⚠️ I'm concerned that this might have unintended consequences. Have you audited all package managers to ensure they'll still be supported when this merges?

Outside of the scope of this work, but `#supported?`, with a default of `true` would be a little easier _for me_ to reason about 😅

Reply:
Bundler was the first package manager to implement this feature, and I initially added the `unsupported?` method, though it wasn’t used at the time since it was part of a future task. For other package managers, we assume the version is supported by default, unless the `unsupported?` condition is explicitly implemented in their respective versions.